src/components/filterdialog/filterdialog.template.html

```diff
@@ -140,4 +140,9 @@
         <div class="collapseContent filterOptions">
         </div>
     </div>
+
+    <button is="emby-button" type="button" class="btnResetAllFilters hide emby-button raised">
```

Comment:
`emby-button` will be added by the custom element.
```suggestion
    <button is="emby-button" type="button" class="btnResetAllFilters hide raised">
```

Reply:
done in cd152f20334107bf31eb61cadf991789757142a2